eo2js/test/resources/transpile/packs/bindings-to-js.json

```diff
@@ -11,7 +18,7 @@
       "args": [
         [
           "        ret = applied(ret, {",
-          "          0: ret_1",
+          "          'α0': ret_1",
```

Comment:
@konard such behaviour would be incorrect. For alpha attributes we should see numbers in result JS files